### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by RomanHrytsiuk**
https://romanhrytsiuk.github.io/layout_search-bar-airbnb/
https://romanhrytsiuk.github.io/layout_search-bar-airbnb/report/html_report/

### CODE REVIEW

**Review by liliia-livandovskaya — CHANGES_REQUESTED** [review]
Remove this blue border Screenshot 2022-11-10 at 16 11 01

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,42 @@
...
+      class="largeForm"
+      action="/"
+      method="post"
+      data-qa="big"
+    >
+      <img
+      class="firstImage"
+      src="./images/Search.svg"
+      alt="search"
+      >
```
Comment (liliia-livandovskaya) [inline src/index.html:24]:
```suggestion
      <img
        class="firstImage"
        src="./images/Search.svg"
        alt="search"
      >
```
fix code style

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,42 @@
...
+      action="/"
+      method="post"
+      data-qa="small"
+    >
+      <img
+      src="./images/Search.svg"
+      alt="search"
+      class="secondImage"
+      >
+      <input
```
Comment (liliia-livandovskaya) [inline src/index.html:45]:
```suggestion
      >
      
      <input
```
don't forget to add space between one-level elements

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+  height: 42px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding-left: 33px;
+}
+
+.smallInput::placeholder {
+  font-style: normal;
```
Comment (liliia-livandovskaya) [inline src/style.css:84]:
```suggestion
```
its default, remove it

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
```
Comment (liliia-livandovskaya) [inline src/style.css:19]:
```suggestion
html {
```
font should be set in `html {...}` tag

**Review by liliia-livandovskaya — CHANGES_REQUESTED** [review]
Great job! Pay attention to the comments

File: src/index.html [hunk 5]
```diff
@@ -11,6 +11,43 @@
...
+        src="./images/Search.svg"
+        alt="search"
+        class="secondImage"
+      >
+      
+      <input
+        type="text"
+        class="smallInput"
+        placeholder="Try &quot;Los Angeles&quot"
+      >
```
Comment (stanislav-topikha) [inline src/index.html:50]:
```suggestion
    <form
      class="form"
      action="/"
      method="post"
      data-qa="big"
    >
      <img
        class="image"
        src="./images/Search.svg"
        alt="search"
      >

      <input
        type="text"
        class="input"
        placeholder="Try &quot;Los Angeles&quot"
        data-qa="keypress"
      >
    </form>

    <form
      class="form form--small"
      action="/"
      method="post"
      data-qa="small"
    >
      <img
        src="./images/Search.svg"
        alt="search"
        class="image image--small"
      >
      
      <input
        type="text"
        class="input input--small"
        placeholder="Try &quot;Los Angeles&quot"
      >
```
lets make it work like this

**Review by stanislav-topikha — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+
+.input--small {
+  font-weight: 300;
+  font-family: inherit;
+  width: 100%;
+  height: 42px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
```
Comment (stanislav-topikha) [inline src/style.css:80]:
```suggestion
  font-weight: 300;
  height: 42px;
  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
```
recheck repeated styles everywhere, DRY

**Review by stanislav-topikha — CHANGES_REQUESTED** [review]

**Review by stanislav-topikha — APPROVED** [review]
GJ 👍🏻

**Pull request closed without merging**